src/util/ChartUtils.ts

```diff
@@ -433,26 +442,26 @@ const isErrorBarRelevantForAxis = (layout?: LayoutType, axisType?: AxisType, dir
 };
 
 export const getDomainOfErrorBars = (
-  data: any[],
-  item: any,
-  dataKey: any,
+  data: Array<object>,
+  item: ReactElement,
+  dataKey: DataKey<any>,
   layout?: LayoutType,
   axisType?: AxisType,
-) => {
+): SpecificDomain | null => {
   const { children } = item.props;
   const errorBars = findAllByType(children, ErrorBar).filter(errorBarChild =>
     isErrorBarRelevantForAxis(layout, axisType, errorBarChild.props.direction),
   );
```

Comment:
We could consider refactoring this function, such that errorBars are passed in. Then the tests would also not need to mock this function.

What do you think?

Reply:
💯 absolutely. I think some natural refactoring opportunities will pop up as we add more types. Just not in this PR 😬